lib/fontcustom/scripts/generate.py

```diff
@@ -126,7 +126,12 @@ def createGlyph( name, source, code ):
     manifest['fonts'].append(fontfile + '.woff')
 
     # Convert EOT for IE7
-    subprocess.call('python ' + scriptPath + '/eotlitetool.py ' + fontfile + '.ttf -o ' + fontfile + '.eot', shell=True)
+    command_args = scriptPath + '/eotlitetool.py ' + fontfile + '.ttf -o ' + fontfile + '.eot'
+    process = subprocess.run('python ' + command_args, shell=True)
+    # Catch command not found error for python vs python3
+    if process.returncode == 127:
```

Comment:
127 is the shell's "command not found" error code, and then the below call has the `check=True` arg added which forces the return status to be 0 else it raises an exception.

Thoughts on changing this to be just returncode != 0?

Reply:
I wonder if we should just default to python3? Otherwise yeah, I'd agree != 0 so that if there's any other problems it also tries this.

Reply:
I'm good with swapping to 0 then!

Reply:
No longer exists! Just swapped default to python3 as fontforge ships with 3.11 on brew currently